parsl/executors/high_throughput/interchange.py

```diff
@@ -475,10 +483,10 @@ def process_tasks_to_send(self, interesting_managers: Set[bytes], monitoring_rad
             len(self._ready_managers)
         )
 
-        if interesting_managers and not self.pending_task_queue.empty():
+        if interesting_managers and (len(self.pending_task_queue) != 0):
             shuffled_managers = self.manager_selector.sort_managers(self._ready_managers, interesting_managers)
 
-            while shuffled_managers and not self.pending_task_queue.empty():  # cf. the if statement above...
+            while shuffled_managers and (len(self.pending_task_queue) != 0):  # cf. the if statement above...
```

Comment:
`SortedList`, like many other Python datastructures, implements `__bool__()`.  Thus, can use truthiness of the data structure directly.  Consider:
```python
if interesting_managers and self.pending_task_queue:
    ...
    while shuffled_managers and self.pending_task_queue:
```